docs/190-paas-operations/002-backup-and-restore.md

```diff
@@ -14,138 +14,48 @@ headerIcon: "paas"
-    - Storage Account Name
-    - Blob Container Name
-    - Storage Resource Group
-
-    The following image shows an example of the parameters.
-
-    ![backup-restore-5.png](../../assets/images/ama/howtos/backup-restore-5.png)
-
-After you executed the backup extension configuration, make sure that the resources were deployed successfully using the following steps:
-
-1. Go to the respective AKS cluster and then do the following: 
-
-    1. In the left pane, under **Settings**, select **Extensions + applications**. You should see the following extension: **azure-aks-backup**
-
-        ![backup-restore-3.png](../../assets/images/ama/howtos/backup-restore-3.png)
-
-    1. In the left pane, under **Settings**, select **Backup (preview)**. You should the following extension: **cluedin-backup-instance**.
-
-        ![backup-restore-4.png](../../assets/images/ama/howtos/backup-restore-4.png)
-
-1. Go to the resource group, and then open the backup vault. Set the **Datasource type** filter to **Kubernetes Services (Preview)**. You should see your backup jobs and instances.
-
-    ![backup-restore-7.png](../../assets/images/ama/howtos/backup-restore-7.png)
-
-## Restore a backup instance
-
-After you configured the backup solution, you can restore any backup instance when needed. The restoration process takes 3–5 minutes.
-
-**To restore a backup instance**
-
-1. Go to the resource group, and then open the backup vault.
-
-1. In the left pane, select **Backup instances**. Then, at the top of the page, select **Restore**.
-
-    The **Restore** dialog opens for you to provide the details.
-
-1. Select the **Backup instance**. Then, in the lower-left corner, select **Next: Restore point**.
-
-    ![backup-restore-8.png](../../assets/images/ama/howtos/backup-restore-8.png)
-
-1. Select the **Restore point** from the list of available backups. Then, in the lower-left corner, select **Next: Restore parameters**.
-
-    ![backup-restore-9.png](../../assets/images/ama/howtos/backup-restore-9.png)
-
-1. On the **Restore parameters** tab, do the following:
-
-    1. Select the target **Kubernetes service** from the list of available services for your subscription.
-
-    1. Select the **Resources to restore**. You can choose to restore the state, the data, or both (complete backup).
-
-        ![backup-restore-11.png](../../assets/images/ama/howtos/backup-restore-11.png)
-
-    1. Validate the restore parameters.
-
-        ![backup-restore-12.png](../../assets/images/ama/howtos/backup-restore-12.png)
-
-    1. In the lower-left corner, select **Next: Review + restore**.
-
-1. Review the restore details, and then select **Restore**.
-
-    ![backup-restore-13.png](../../assets/images/ama/howtos/backup-restore-13.png)
-
-    You'll receive a notification when the restore is completed.
+# Further information
+If you require further information at any point regarding the above, please reach out to CluedIn support who will assist.
```

Comment:
I think this should be deleted. CluedIn support is mentioned in several places in the article, there is no need to repeat it.

Reply:
Removed